File: Android/GestiMedmedico/app/src/main/java/com/example/gestimedmedico/Paciente.kt
package com.example.gestimedmedico

import java.util.*
import kotlin.collections.ArrayList

class Paciente(sip: String, nombre: String, apellidos: String, numContacto: String, nacimiento: Date, medicos: List<Medico>, urgencia: String, tratamiento: Tratamiento, habitacion: Habitacion) {
    var sip: String
    var nombre: String
    var apellidos: String
    var numContacto: String
    var nacimiento: Date
    var medicos: List<Medico>
    var tratamiento: Tratamiento
    var urgencia : String
    var habitacion: Habitacion

    init {
        this.sip = sip
        this.nombre = nombre
        this.apellidos = apellidos
        this.numContacto = numContacto
        this.nacimiento = nacimiento
        this.medicos = medicos
        this.tratamiento = tratamiento
        this.urgencia = urgencia
        this.habitacion = habitacion
    }
}

File: Escritorio/Proyecto/Proyecto/clases/Paciente.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto.clases
{
    class Paciente
    {
        private string sip;
        private string nombre;
        private string apellidos;
        private string numContacto;
        private string dolencias;
        private string urgencia;
        private DateTime nacimiento;
        private Collection<Medico> medicos;
        private Tratamiento tratamiento;

        //Constructores
        public Paciente() { }

        public Paciente(string sip, string nombre, string apellidos, string numContacto, DateTime nacimiento, string dolencias, string urgencia)
        {
            this.sip = sip;
            this.nombre = nombre;
            this.apellidos = apellidos;
            this.numContacto = numContacto;
            this.nacimiento = nacimiento;
            this.dolencias = dolencias;
            this.urgencia = urgencia;
        }

        // Getters
        public string getSip() { return sip; }
        public string getNombre() { return nombre; }
        public string getApellidos() { return apellidos; }
        public string getNumContactos() { return numContacto; }
        public DateTime getNacimiento() { return nacimiento; }
        public Collection<Medico> getMedicos() { return medicos; }
        public Tratamiento getTratamiento() { return tratamiento; }

        //Setters
        public void setSip(string sip) { this.sip = sip; }
        public void setNombre(string nombre) { this.nombre = nombre; }
        public void setApellidos(string apellidos) { this.apellidos = apellidos; }
        public void setNumContacto(string numContacto) { this.numContacto = numContacto; }
        public void setNacimiento(DateTime nacimiento) { this.nacimiento = nacimiento; }
        public void setMedicos(Collection<Medico> medicos) { this.medicos = medicos; }
        public void setTratamiento(Tratamiento tratamiento) { this.tratamiento = tratamiento; }
    }
}
